app/models/org_space_policy.rb

```diff
@@ -27,8 +37,28 @@ def ensure_exists
     ensure_space_layer
   end
 
+  def create
+    load_policy(template_create_org_space)
```

Comment:
do we understand what will happen if this policy already exists when this `load_policy` command is run? for ex, if they provision a service instance in a space, then deprovision with preserve_policy on, then re-provision - what will happen? worth adding a test for this at least imo